app/services/notification/AwsCredentials.scala

```diff
@@ -0,0 +1,14 @@
+package services.notification
+
+import javax.inject.Singleton
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
+
+@Singleton
+class AwsCredentials {
```

Comment:
Provider is (essentially) static here so you can inline this without affecting anything. You'll see in (e.g.) `metrics.CloudWatch` how it's done elsewhere. Magic Dependency Injection makes it super easy to create 1000 classes where 1 will do, it takes some discipline to find the right balance between flexibility and simplicity. I prefer simplicity, especially in "microapps". Other opinions exist and are valid :-)